R/dfm.R

```diff
@@ -383,7 +384,9 @@ make_null_dfm <- function(feature = NULL, document = NULL) {
         i = NULL,
         j = NULL,
         dims = c(length(document), length(feature))
-    ), "dgCMatrix")
+    ), "CsparseMatrix") %>%
```

Comment:
Please do not use pipes in the code for me. I use them in the global environment but never in packages.